public/locales/en/geolocationpanel.json

```
@@ -0,0 +1,41 @@
+{
+  "geolocation": {
+    "title": "Geo Location",
+    "select-anchor-label": "Select an anchor",
+    "no-anchor-panel": "Currently there is no data for locations on {{anchor}}"
+  },
```

same comment here. What naming scheme should we use for the actual panel component?   Maybe
```suggestion
  "panel": {
    "title": "Geo Location",
    "select-anchor-label": "Select an anchor",
    "no-anchor-panel": "Currently there is no data for locations on {{anchor}}"
  },
```
or just remove the namespace completely?
```suggestion
  "title": "Geo Location",
  "select-anchor-label": "Select an anchor",
  "no-anchor-panel": "Currently there is no data for locations on {{anchor}}",
```